exec/jdbc/pom.xml

```diff
@@ -100,6 +100,11 @@
       <groupId>xalan</groupId>
       <artifactId>xalan</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Might not be needed. I don't think Drill depends directly on it (but only transitively through Avatica)